cdap-ui/app/features/streams/templates/detail.html

```diff
@@ -16,6 +16,11 @@
       </button>
       <ul class="dropdown-menu" role="menu">
         <li>
+          <a href="" ng-click="sendEvents()">
```

Comment:
Does it make sense to have this button on the Status page? Like you can just click on a stream on the flow page to ingest?